include/zephyr/data/cobs.h

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2024 Kelly Helmut Lord
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DATA_COBS_H_
+#define ZEPHYR_INCLUDE_DATA_COBS_H_
+
+#include <stddef.h>
+#include <sys/types.h>
+#include <zephyr/sys/util.h>
+#include <zephyr/net_buf.h>
+#ifdef __cplusplus
```

Comment:
Add some breathing room here.

```suggestion
#include <zephyr/net_buf.h>

#ifdef __cplusplus
```